balancer-js/src/lib/utils/generate-address-books.ts

```diff
@@ -0,0 +1,172 @@
+import axios from 'axios';
+import { Network } from '@/types';
+import _ from 'lodash';
+import * as fs from 'fs';
+
+type AddressDictByNetwork = {
+  [key: string]: {
+    contracts: AddressDict;
+    tokens: AddressDict;
+  };
+};
+
+type AddressDict = {
+  [key: string]: string;
+};
+
+const addressBookUrl =
+  'https://raw.githubusercontent.com/BalancerMaxis/bal_addresses/main/outputs/addressbook.json';
+
+const developAddressBookOutputUrl =
```

Comment:
I wanted to test running `yarn address-book:generate`. Currently `develop` doesn't have the address.json. I changed to use:
`const developAddressBookOutputUrl =
  'https://github.com/balancer/balancer-sdk/blob/improve-network-deployment/balancer-js/src/lib/constants/addresses.json';`
instead but script is throwing:
```
Comparing Contracts of network:  MAINNET
/Users/jg/Documents/sdk-bruno/balancer-js/src/lib/utils/generate-address-books.ts:133
  const developContractsKeys = Object.keys(develop.contracts);
                                      ^
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at compareOutputWithDevelop (/Users/jg/Documents/sdk-bruno/balancer-js/src/lib/utils/generate-address-books.ts:133:39)
    at /Users/jg/Documents/sdk-bruno/balancer-js/src/lib/utils/generate-address-books.ts:83:9
    at Array.map (<anonymous>)
    at generateAddressesFile (/Users/jg/Documents/sdk-bruno/balancer-js/src/lib/utils/generate-address-books.ts:43:6)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    ```
    Anything I should try differently?

Reply:
Let me take these steps to see what's going on

Reply:
The url for this case should be: https://raw.githubusercontent.com/balancer/balancer-sdk/improve-network-deployment/balancer-js/src/lib/constants/addresses.json
In order to fetch the raw content

Reply:
Thanks!